website/docs/r/acm_certificate.html.markdown

```diff
@@ -11,21 +11,44 @@ description: |-
 The ACM certificate resource allows requesting and management of certificates
 from the Amazon Certificate Manager.
 
-It deals with requesting certificates and managing their attributes and life-cycle.
+ACM certificates can be created in three ways:
+Amazon-issued, where AWS provides the certificate authority and automatically manages renewal;
+imported certificates, issued by another certificate authority;
+and private certificates, issued using an ACM Private Certificate Authority.
+
+## Amazon-Issued Certificates
+
+For Amazon-issued certificates, this resource deals with requesting certificates and managing their attributes and life-cycle.
 This resource does not deal with validation of a certificate but can provide inputs
-for other resources implementing the validation. It does not wait for a certificate to be issued.
+for other resources implementing the validation.
+It does not wait for a certificate to be issued.
 Use a [`aws_acm_certificate_validation`](acm_certificate_validation.html) resource for this.
 
 Most commonly, this resource is used together with [`aws_route53_record`](route53_record.html) and
 [`aws_acm_certificate_validation`](acm_certificate_validation.html) to request a DNS validated certificate,
 deploy the required validation records and wait for validation to complete.
 
-Domain validation through email is also supported but should be avoided as it requires a manual step outside
-of Terraform.
+Domain validation through email is also supported but should be avoided as it requires a manual step outside of Terraform.
 
 It's recommended to specify `create_before_destroy = true` in a [lifecycle][1] block to replace a certificate
 which is currently in use (eg, by [`aws_lb_listener`](lb_listener.html)).
 
+## Imported Certificates
```

Comment:
Use of a heading starting with `Import` triggers this `tfproviderdocs` rule - https://github.com/bflad/tfproviderdocs/blob/23ecf1a2541c471a75aee8d2e7607b20ea4e23d3/check/contents/sections.go#L188 - with an error reported:

```
Error checking Terraform Provider documentation: 1 error occurred:
	* website/docs/r/acm_certificate.html.markdown: error checking file contents: import section heading (Imported Certificates) should be: Import
```

Reply:
I've created bflad/tfproviderdocs#49 to address it